src/Components/Common/EditUser.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Controller, Control } from 'react-hook-form';
+import { IEditUserForm } from '../../common/interfaces';
+import bg from '../../images/Edit-User-bg.svg';
+
+interface IEditUserInput {
+  name:'userName' | 'score';
+  control: Control<IEditUserForm> | undefined;
+  placeholder:string;
+  type:string;
+}
```

Comment:
interfaces and types should be in another file. You should divide the styles and logical components of your app